bfvmm/include/vmcs/vmcs_intel_x64.h

```diff
@@ -294,6 +299,8 @@ class vmcs_intel_x64
 
     friend class vmcs_ut;
 
+    bitmap *m_msr_bitmap;
```

Comment:
I was getting bitched at by the compiler at one point about it's initialization as an object, but changing it now seems fine.